bin/replication.js

```diff
@@ -115,8 +114,10 @@ class _SetupReplication {
             'default' : sourceProfile;
         const verifyTargetProfile = targetProfile === undefined ?
             'default' : targetProfile;
-        source.auth.vault.host = 'localhost';
-        source.s3.host = 'localhost';
+
+        // TODO: Federation fix made #1125?
+        // source.auth.vault.host = 'localhost';
+        // source.s3.host = 'localhost';
         const destHost = this.destHosts.pickHost().host;
```

Comment:
OK, I understand, I think it deserves a comment explaining the reason though, and/or a FIXME for later.